ros_buildfarm/aggregate_status.py

```diff
@@ -0,0 +1,262 @@
+import os
+import re
+import yaml
+import requests
+import collections
+from .config import get_index as get_config_index
+from .config import get_release_build_files
+from .status_page import REPOS_DATA_NAMES as APT_REPOS
+
+"""
+Aggregate Status takes the yaml generated by the different CPU architectures (e.g. ros_lunar_default, ros_lunar_ds)
+and combines them into one yaml file per ROS distro.
+It also creates an English description of each package across all of the architectures.
+
+Example Descriptions:
+ - released - All apt_repos have the same version number
+ - waiting for new release - Build/test have the same version number, but there's nothing for main
+ - waiting for re-release - Build/test have the same version number, but there's an older version for main
```

Comment:
Similar to the original comment: https://github.com/ros-infrastructure/ros_buildfarm/pull/555#issuecomment-404328056

Both of the short labels "waiting for new release" / "waiting for re-release" are misleading to me. Under waiting for release I expect that someone has to do a release which is not the case here. Both are merely waiting for a sync to happen to make the latest release available in the `main` repo.